runtime/service.go

```diff
@@ -67,6 +67,7 @@ const (
 	defaultVsockPort        = 10789
 	minVsockIOPort          = uint32(11000)
 	firecrackerStartTimeout = 5 * time.Second
+	stopVMTimeoutSeconds    = 5
```

Comment:
nit: For this variable, I think it should be a `time.Duration` (so just `5 * time.Second`) so it can stay agnostic of particular units.

Also, can this be named more like `defaultStopVMTimeout`? Just to make it clear it's a default value that can be optionally overridden.